services/QuillLMS/spec/controllers/admin_access_controller_spec.rb

```diff
@@ -70,15 +70,16 @@
   describe '#request_upgrade_to_admin_from_existing_admins' do
     it 'creates an admin info record with the requisite information, an admin approval request record, and fires the necessary segment workers' do
       reason = 'I really want to be an admin.'
+      new_user = [true, false].sample
```

Comment:
I like the impulse to not repeat all these expectations for both `true` and `false`, and from a practical perspective, this should be just fine.  However, it is possible (if statistically unlikely) that only either `true` or `false` paths are actually working and that `sample` just keeps returning the working value.  I think you can make sure that both branches run every test with a small tweak:
```ruby
[true, false].each do |new_user|
  # put all your test code from line 74 to 82 inside this block, and it'll run twice, once for each value
end
```